run

```diff
@@ -122,7 +122,13 @@ psql() {
   # Connect to PostgreSQL
   # shellcheck disable=SC1091
   . .env
-  _dc postgres psql -d "${POSTGRES_DB}" -U "${POSTGRES_USER}" "${@}"
+
```

Comment:
You can simplify everything down to `-d "${POSTGRES_DB:-"${POSTGRES_USER}"}"` btw. That will use the DB var if it exists else fall back to the user which is guaranteed to be defined.